### DISCUSSION TITLE
why Adoptium have different name for JDK 21 LTS if downloaded from release page and if installed it from repo configuration

### DISCUSSION THREAD

**Discussion by pandeyv-del** [adoptium/adoptium-support#1028] [Q&A] [answered]
observed Adoptium configure different name if we download it directly from release page [https://adoptium.net/en-GB/temurin/releases/](url) and if we configure this through repos like [https://adoptium.net/en-GB/installation/linux/](url)

first unzip it with this **jdk-21.0.1+12** while later configure itself as **tremium21** wanted to know what is difference between them.

**Comment by karianna** [comment, marked as answer]
They are the same binary and just have different file names to conform with the OS/package manager naming stds.

**Reply by pandeyv-del** [reply]
Yes, but sometimes it causes issues specifically on an automated process. Should we not follow the same name on the same OS as you mentioned for OS specific naming standards? for example: if it is RPM based OS it would have `xyz` name irrespective of how the installation has happened etc.

**Reply by karianna** [reply]
I think changing naming at this stage would break a lot of other folks who are used to the naming convention we have now.
